CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ With this release InfluxDB is moving to Go 1.5.
 - [#3876](https://github.com/influxdb/influxdb/pull/3876): Allow the following syntax in CQs: INTO "1hPolicy".:MEASUREMENT
 - [#3975](https://github.com/influxdb/influxdb/pull/3975): Add shard copy service
 - [#3986](https://github.com/influxdb/influxdb/pull/3986): Support sorting by time desc
+- [#3930](https://github.com/influxdb/influxdb/pull/3930): Wire up TOP aggregate function
```

Comment:
Duplicating this in the CHANGELOG looks wrong to me. I think it's better to keep the first line, and then add "fixes [#1821](https://github.com/influxdb/influxdb/issues/1821)" to the line.